packages/slate/src/constants/operation-attributes.js

```diff
@@ -29,6 +29,7 @@ const OPERATION_ATTRIBUTES = {
     'value',
```

Comment:
property called 'original' added to the merge_node and split_node operations

Reply:
While I was writing documentation, I concluded that the name 'original' is bad here. I'm not sure what a good name is, so I'll wait for your feedback before I go pick one :)

I think maybe if we just have `type` and `data` that's clear, easy to document, and probably more lightweight